src/components/BlogList.jsx

```diff
@@ -2,24 +2,49 @@ import BlogPost from "./BlogPost";
 import Pagination from "./Pagination";
 import React from "react";
 import blogs from "../data/blogs.json";
+//-----imported useState.
```

Comment:
Be careful to overuse comments, they should be there to explain what is not obvious in the code.